### DISCUSSION TITLE
NFT not visible

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#364] [Q&A] [answered]
## Issue
Today morning I solved the `Lesson 1 NFT Challenge`. It's already been more than `8 hours` but the NFT is still not visible. 
If anyone knows the issue please do let me know.

## Transaction 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/93525771/05120017-51ce-451a-ba81-cbc84ee223e9)
## NFT not visible
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/93525771/0086c46c-85f0-4ff8-a693-98e16fcb30ba)

**Comment by Naman1729** [comment, marked as answer]
Hey @edsondsouza , you have to import your NFT in metamask by follow these steps :

1. Open Metamask **Activity section**.
2. Click on **Contract Interactions**.
3. Click on **View on block explorer** .Then , new tab is open .
4. Go to **ERC-721 Tokens Transferred**: and Click on **NFT** .
5. Copy the **Contract Address and Token ID**.
6. Open Metamask **NFT section**.
7. Click on **import NFT**.
8. Paste **Contract Address and Token ID**.
9. Click on **ADD**.

**Reply by ghost** [reply]
Awesome, got it. Thank You!
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/93525771/f170b7f6-e556-4acd-90a1-4c72e51cbb7c)

**Reply by TSHEPO-CLOUD** [reply]
Hey @edsondsouza i hope you doing well, am not sure if am doing anything wrong interacting with a contract on etherscan, as i get this `Transaction decoding is not available for chainId 11155111` error on Lesson one Challenge, if you have any idea as to what is wrong  please assist

**Reply by ghost** [reply]
Heya 👋🏻 @TSHEPO-CLOUD I'm doing good. How are you?

Since it's a challenge I can't help you much and I've never come accross such `transacton chainId error`. But I'll give you some hints that might help you solve the challenge.

- check if you have enough eth to pay for the gas price.
- connect your wallet using different account (eg: Account 2, Account 3 etc)

Hope that helps. Sorry I can't help you more than this.

Don't tell this to anyone; if you go deeper into the issues section you might find something interesting 🤫😁

**Reply by PatrickAlphaC** [reply]
boom :)

**Reply by TSHEPO-CLOUD** [reply]
Thanks I managed to solve the problem

**Reply by mrMorningstar96** [reply]
Hey I followed steps you recommended, my NFT's are imported but I cannot see picture of them (check image below).
Also I tried this :
Go to meta mask, go to settings -->> advanced --- >> clear meta mask logs.
Then go to security and privacy, enable autoDetectNft and openSea api. (but that either did not workout for me)
@Naman1729 @PatrickAlphaC 
![nft error](https://github.com/Cyfrin/foundry-full-course-f23/assets/139367140/fa0b33d2-7ece-445f-b107-9df5e0adb3ae)

**Reply by Blockchain-Oracle** [reply]
i have same issues here .. any gist of how you worked it out ?

**Reply by rihadalisunny** [reply]
@LukaNikolic96 @Blockchain-Oracle Hey just remove the nfts and reenter them after you changed those settings above and make sure to also turn on the Display NFT option in the metamask settings.

You can get the contract address and the ID if you click on the NFT in metamask. Hope it helps.

**Comment by mrMorningstar96** [comment]
> i have same issues here .. any gist of how you worked it out ?
Unfortunately I still did not resolve this issue.
